ReactiveCocoaTests/InterceptingSpec.swift

```diff
@@ -76,13 +76,25 @@ class InterceptingSpec: QuickSpec {
 				for _ in 1 ... 10 {
 					var isDeadlocked = true
 
-					DispatchQueue.global(priority: .high).async {
-						_ = object.reactive.trigger(for: #selector(object.increment))
+					if #available(*, macOS 10.10) {
+						DispatchQueue.global(qos: .userInitiated).async {
+							_ = object.reactive.trigger(for: #selector(object.increment))
+
+							DispatchQueue.global(qos: .userInitiated).async {
+								_ = object.reactive.trigger(for: #selector(object.increment))
 
+								isDeadlocked = false
+							}
+						}
+					} else {
 						DispatchQueue.global(priority: .high).async {
 							_ = object.reactive.trigger(for: #selector(object.increment))
 
-							isDeadlocked = false
+							DispatchQueue.global(priority: .high).async {
+								_ = object.reactive.trigger(for: #selector(object.increment))
+
+								isDeadlocked = false
+							}
```

Comment:
These two things seem mostly the same. It looks like only the queue is different. So we should extract that and reuse everything else.

```swift
let queue: DispatchQueue
if #available(*, macOS 10.10 ) {
    queue = ...
} else {
    queue = ...
}
queue.async { ... }
```